packages/drivers/odsp-driver-definitions/src/tokenFetch.ts

```diff
@@ -11,6 +11,17 @@ export interface TokenResponse {
 	/** Token value */
 	token: string;
 
+	/**
+	 * Authorization header value that must be passed verbatim when making network call that requires the token.
+	 * If not returned then it is caller's responsibility to generate Authorization header value based on 'token' value.
+	 * For that caller needs to detect the type of returned token and generate the header value appropriately.
```

Comment:
Is it really "detect"? Or is it - it's bearer by default?

Reply:
I will modify the comment. On this interface, it would be assumed `Bearer`.